modelon/impact/client/operations/workspace/exports.py

```diff
@@ -0,0 +1,104 @@
+import os
+from modelon.impact.client.sal.service import Service
+from modelon.impact.client.operations.base import AsyncOperation, AsyncOperationStatus
+from modelon.impact.client import exceptions
+
+
+class BinaryBlob:
```

Comment:
BinaryBlob -> Export, would be more clear. It is not representing a binary blob it is representing an URL to something that is exported and ready for download.